app/controllers/concerns/hyrax/manages_embargoes.rb

```diff
@@ -0,0 +1,21 @@
+module Hyrax
+  module ManagesEmbargoes
+    extend ActiveSupport::Concern
+
+    included do
+      include Hyrax::ThemedLayoutController
+      with_themed_layout '1_column'
```

Comment:
@darrenleeweber The install generator sets a default for Hyrax-based applications by adding that code to the app's `ApplicationController`. It'd be worth removing both lines and seeing what effect that has on the UI.